Tasks/IISWebAppManagement/make.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Can you check whether there's a way to bundle appcmd.exe along with the task. If Yes, then should we do it ?